test/server/cards/04-MM/Keyfrog.spec.js

```diff
@@ -65,5 +69,96 @@ describe('Keyfrog', function () {
 
             expect(this.keyfrog.location).toBe('discard');
         });
+
+        it("should end the game when it is the last key and controller's turn", function () {
```

Comment:
Should the test name be "should end the game when it is the last key and **not** controller's turn"?

Reply:
Fixed